app/assets/javascripts/application/custom-editor.js

```diff
@@ -0,0 +1,21 @@
+var mumuki = mumuki || {};
+
+(function (mumuki) {
+
+  var CustomEditor = {
+    sources: [],
+
+    addSource: function (source) {
+      CustomEditor.sources.push(source);
+    },
+
+    // Each external source must implement getContent method
+
+    getContent: function () {
```

Comment:
This change is backward-compatible, isn't it?

Reply:
Yep!